Packs/Troubleshoot/IncidentFields/incidentfield-Instance_Name.json

```diff
@@ -0,0 +1,34 @@
+{
+ "associatedToAll": true,
```

Comment:
i think all fields should not be associated to all types, but only the troubleshooting ones